src/2019/components/Footer.js

```diff
@@ -34,6 +36,14 @@ const ContactHeader = styled.h1`
     line-height: 1.7;
 `
 
+const ContactLink = styled.a`
+    color: ${LIGHT_BLUE};
+
+    &:hover {
```

Comment:
don't see why the hover is needed here. color: `${LIGHT_BLUE} !important` should do the trick on line 40